Makefile

```diff
@@ -16,7 +16,10 @@ develop: clean-pyc
 	pip install -e .[testing,docs]
 	npm install --no-save && npm run build
 
-lint:
+spelling:
+	codespell -S *.po,*.map,*/vendor/*,*vendor.js*,./node_modules/*,./docs/_build/*,./storybook-static/*,./wagtail/admin/static/wagtailadmin/js/*,./package-lock.json
+
+lint: spelling
```

Comment:
This change will now additionally run linting as part of the CircleCI task. Is that something we want?